src/Controller/Category/FindAllController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Category;
+
+use App\Entity\Category;
+use Symfony\Component\DependencyInjection\Container;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+class FindAllController
+{
+    public function __invoke(Request $request, Container $container)
+    {
+        $entityManager = $container->get('entity.manager');
+        $categories = $entityManager->findAll(Category::class);
+
+        return new JsonResponse([
+            'Data' => $categories
```

Comment:
to fit an API 'categories' instead of data would have been better.
Speaking of API, I recommend this article to improve your skills in REST
https://martinfowler.com/articles/richardsonMaturityModel.html